docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -13,10 +13,9 @@ permalink: /FHIRServerUsersGuide/
   * [2.2 Upgrading an existing server](#22-upgrading-an-existing-server)
   * [2.3 Docker](#23-docker)
 - [3 Configuration](#3-configuration)
-  * [3.1 Encoded passwords](#31-encoded-passwords)
-  * [3.2 Property names](#32-property-names)
-  * [3.3 Tenant-specific configuration properties](#33-tenant-specific-configuration-properties)
-  * [3.4 Persistence layer configuration](#34-persistence-layer-configuration)
+  * [3.1 Liberty server configuraiton](#31-liberty-server-configuration)
```

Comment:
```suggestion
  * [3.1 Liberty server configuration](#31-liberty-server-configuration)
```

Reply:
good catch, fixed in a subsequent commit